src/Look/TrafficLook.hpp

```diff
@@ -7,20 +7,68 @@
 #include "ui/canvas/Pen.hpp"
 #include "ui/canvas/Brush.hpp"
 #include "ui/canvas/Icon.hpp"
+#include "FLARM/TrafficClimbAltIndicators.hpp"
 
 class Font;
 
-struct TrafficLook {
-  static constexpr Color safe_above_color{0x1d,0x9b,0xc5};
-  static constexpr Color safe_below_color{0x1d,0xc5,0x10};
-  static constexpr Color warning_color{0xfe,0x84,0x38};
-  static constexpr Color warning_in_altitude_range_color{0xff,0x00,0xff};
-  static constexpr Color alarm_color{0xfb,0x35,0x2f};
+struct TrafficLook
+{
+  struct colorful_traffic_colors
+  {
+    struct above
+    {
+      static constexpr Color climb_good = {0xff, 0x66, 0x66}; // light red
+      static constexpr Color climb_up = {0xff, 0xff, 0x66};   // light yellow
+      static constexpr Color climb_down = {0x66, 0x66, 0xff}; // light blue
+    };
+
+    struct same
+    {
+      static constexpr Color climb_good = {0xff, 0x00, 0x00}; // red
+      static constexpr Color climb_up = {0xff, 0xff, 0x00};   // yellow
+      static constexpr Color climb_down = {0x00, 0x00, 0xff}; // blue
+    };
+
+    struct below
+    {
+      static constexpr Color climb_good = {0x99, 0x00, 0x00}; // dark red
+      static constexpr Color climb_up = {0x99, 0x99, 0x00};   // dark yellow
+      static constexpr Color climb_down = {0x00, 0x00, 0x99}; // dark blue
+    };
+  };
+
+  struct TrafficLookColor
+  {
+    static constexpr Color above = {0x1d, 0x9b, 0xc5};
+    static constexpr Color same = {0xff, 0x00, 0xff};
```

Comment:
Colors are defined in src/ui/canvas/Color.hpp. There is even a function for darkening and lighting them. Please stick to those definitions. 

Red is not a good color.
Find a different solution to display good climb.

Reply:
green would be probably better anyhow.